pkg/apis/pingcap/v1alpha1/dm_config.go

```diff
@@ -95,3 +100,8 @@ type DMSecurityConfig struct {
 	// +optional
 	CertAllowedCN []string `toml:"cert-allowed-cn,omitempty" json:"cert-allowed-cn,omitempty" yaml:"cert-allowed-cn,omitempty"`
 }
+
+// DM experimental config
```

Comment:
```suggestion
// DM experimental config
// +k8s:openapi-gen=true
```

Reply:
thanks for  review, now dm-master config looks like [this](https://github.com/pingcap/ticdc/blob/9721053edae6b146b726bd5daec04a0dc8c57b1a/dm/dm/master/dm-master.toml#L46)

and this new config only worked in dm-master not affects dm-worker